ctapipe/tools/muon_reconstruction.py

```diff
@@ -75,39 +67,36 @@ class MuonAnalysis(Tool):
 
     aliases = {
         ("i", "input"): "EventSource.input_url",
-        ("o", "output"): "MuonAnalysis.output",
+        ("o", "output"): "DataWriter.output_path",
         ("m", "max-events"): "EventSource.max_events",
         ("t", "allowed-tels"): "EventSource.allowed_tels",
     }
 
     flags = {
-        "f": ({"MuonAnalysis": {"overwrite": True}}, "Overwrite output file"),
+        "f": (
+            {"DataWriter": {"overwrite": True}},
+            "Overwrite output file if it exists",
+        ),
         **flag(
             "overwrite",
```

Comment:
In #2111 we decided to not use `-f` for "forcing overwrite" in tools.

Reply:
We should just remove the tool now...